d2l-icon-button.html

```diff
@@ -18,7 +18,8 @@
 			behaviors: [window.D2L.PolymerBehaviors.IconButtonBehavior],
```

Comment:
If we are using title instead of textContent, then this isn't necessary. Ditto for link (unless we want to keep it for link?)